src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/MediaTransportControls_themeresources.xaml

```diff
@@ -175,7 +175,7 @@
                             </VisualStateGroup>
                             <!-- Media state visual states -->
                             <!-- Audio Selection Button visibility states -->
-                            <VisualStateGroup x:Name="AudioSelectionAvailablityStates">
+                            <VisualStateGroup x:Name="AudioSelectionAvailabilityStates">
```

Comment:
While technically valid, it's an error that comes from WinUI, and we should be keeping the same content as from the original file: https://github.com/microsoft/microsoft-ui-xaml/blob/20dcc18d040aae93f3d464cc96b0023783cdcf73/controls/dev/CommonStyles/MediaTransportControls_themeresources.xaml#L167